application/holder/src/service/cli/open.ts

```diff
@@ -34,25 +34,23 @@ export class Action extends CLIAction {
         return this.error;
     }
 
-    public execute(cli: Service): Promise<void> {
+    public async execute(cli: Service): Promise<void> {
```

Comment:
Please also make sure, all places of this function's usage are checked and corrected also. It's quite radical change (sync -> async) and we should make sure - all calls of changed functions are corrected too.